utils/urls.js

```diff
@@ -0,0 +1,3 @@
+const { PREDICTION_URL, CURRENT_USER } = require('sharify').data
+
+export const liveAuctionUrl = (id) => `${PREDICTION_URL}/${id}${CURRENT_USER ? '/login' : ''}`
```

Comment:
Thanks for adding this :) Per previous discussions in slack and elsewhere I would update the file location path to be grouped by domain: `utils/domain/auctions/urls.js`.